Streetcode/Streetcode.BLL/Mapping/AdditionalContent/TagProfile.cs

```diff
@@ -8,6 +8,6 @@ public class TagProfile : Profile
 {
     public TagProfile()
     {
-        CreateMap<Tag, TagDTO>().ReverseMap();
+        CreateMap<Tag, TagDTO>().ForMember(x => x.Streetcodes, conf => conf.Ignore());
```

Comment:
Why we should ignore streetcodes p-ty mapping?

Reply:
Because we have infinite loop in our dto.

Reply:
If that's the case, what you've written is a temporary stub. I am sure there exists a way to map all properties correctly (consider reviewing other profiles addressing this issue).

Reply:
During initial db-setup, I also encoutered infinite cycles in model relationships, however, I managed to find a way to fix the latter.

Reply:
done